src/game/Box.js

```diff
@@ -8,14 +8,27 @@ const BoxContainer = styled('div')(
         padding: '2px',
         margin: '2px',
     },
-    ({ boxSize, winningBox, selected, clickable, context }) => ({
+    ({
```

Comment:
This style is too complex. It would be better to apply dedicated classes depending on the props.